packages/dashboard/src/components/sidePanel/sections/textSettingSection/text.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { FC, MouseEventHandler } from 'react';
+import { DashboardMessages } from '../../../../messages';
+import { useTextWidgetInput } from '../../utils';
+import { ExpandableSection, Grid, Select, SelectProps } from '@cloudscape-design/components';
+import ColorPicker from '../../shared/colorPicker';
+import { fontFamilyBase, fontFamilyMonospace } from '@cloudscape-design/design-tokens';
+import { NonCancelableEventHandler } from '@cloudscape-design/components/internal/events';
+import './index.scss';
+
+export type TextComponentProps = {
+  messageOverride: DashboardMessages;
+};
+
+const ButtonWithState: FC<{ checked: boolean; onToggle: MouseEventHandler }> = ({ checked, children, onToggle }) => {
+  return (
+    <div className={`text-style-button ${checked ? 'checked' : ''}`} onClick={onToggle}>
+      {children}
+    </div>
+  );
+};
+
+const fontLabelMap: { [font: string]: string } = {
+  [fontFamilyBase]: 'Base font',
+  [fontFamilyMonospace]: 'Mono Space',
+  unset: 'Unset',
+};
+const getFontLabel = (font: string) => {
+  return fontLabelMap[font] || font;
+};
+
+const TextSettings: FC<TextComponentProps> = ({ messageOverride }) => {
+  console.log(fontFamilyBase, fontFamilyMonospace);
```

Comment:
delete

Reply:
right. missed this one. removing it now.